src/tests/handlers/trial.handler.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+import { getTrialById } from '../../controllers/database/handlers/getTrialById.handler';
+import { Trial } from '../../entities/trial.entity';
+import { HandlerTestBase } from '../utils/handlerTestBase';
+import { getTrials } from '../../controllers/database/handlers/getTrials.handler';
+import { SearchQuery } from '../../contracts/search.query';
+import { EntityManager, PostgreSqlDriver } from '@mikro-orm/postgresql';
+
+describe('Handler tests', () => {
+  describe('trial Handler tests', () => {
+    let base = new HandlerTestBase();
+    let em: EntityManager<PostgreSqlDriver>;
+
+    after(async () => base.after());
+    before(async () => ({ base } = await HandlerTestBase.before()));
+
+    beforeEach(async () => {
+      em = await base.beforeEach();
+    });
+    afterEach(() => base.afterEach());
+
+    it('should return the trial when id is asked', async () => {
+      const trial = await em.findOneOrFail(Trial, { id: '123' });
```

Comment:
Just use findOne without specific Id. The test doesn't care which trial it is. That way changing the test data does not make this test fail